packages/components/src/Well/Well.module.css

```
@@ -0,0 +1,222 @@
+}
+
+/* // These are @deprecated but must come after `color` so not everything will default to white
+// This will be @deprecated */
+.default {
+  --background-color: var(--color-gray-200);
+  --border-color: var(--color-gray-500);
+}
+
+.negative {
+  --background-color: var(--color-red-100);
+  --border-color: var(--color-red-500);
+}
+
+.informative {
+  --background-color: var(--color-blue-100);
+  --border-color: var(--color-blue-400);
+}
+
+.cautionary {
+  --background-color: var(--color-yellow-100);
+  --border-color: var(--color-yellow-700);
+}
+
+.assertive {
+  --background-color: var(--color-orange-100);
+  --border-color: var(--color-orange-600);
+}
+
+.prominent {
+  --background-color: var(--color-purple-100);
+  --border-color: var(--color-purple-400);
+}
+
+/* // @deprecated end
+
+// Border Style Variations */
+.solid {
+  --border-style: solid;
+}
+
+.dashed {
+  --border-style: dashed;
+}
+
+.none {
+  --border-color: transparent;
+  --border-style: none;
+}
+
+.aiMoment {
+  --clr-1: rgba(100 210 215 / 100%) 0%;
+  --clr-2: rgba(103 199 175 / 100%) 20%;
+  --clr-3: rgba(100 210 215 / 100%) 41%;
+  --clr-4: rgba(200 136 226 / 100%) 100%;
+  --animated-border-width: 2px;
+
+  background-color: transparent;
+  position: relative;
+  border: 1px solid transparent;
+  border-radius: calc(var(--border-borderless-border-radius));
+}
+
+/* This absolute element is used to give the illusion of a gradient border without expanding the size of the Well */
+.aiMomentBackground {
+  margin: var(--animated-border-width);
+  box-sizing: border-box;
+  position: absolute;
+  content: '';
+  z-index: -1;
+  inset: 0;
+  display: block;
+  border-radius: inherit;
+  border: 1px solid transparent;
+  background-color: var(--background-color);
+}
+
+.aiMoment::before,
+.aiMoment::after {
+  content: '';
+  position: absolute;
+  inset: -1px;
+  z-index: -2;
+  background-image: linear-gradient(
+    var(--gradient-angle),
+    var(--clr-1),
+    var(--clr-2),
+    var(--clr-3),
+    var(--clr-4)
+  );
+  border-radius: 8px;
+  background-size: 100% 100%;
+  background-position: 0 50%;
+  transition:
+    filter 1s ease-in,
+    opacity 1s ease-in;
+}
+
+.aiMoment::before {
+  filter: blur(0);
+  opacity: 0;
+}
+
+.aiMoment::after {
+  opacity: 1;
+}
+
+.aiMomentLoading::after,
+.aiMomentExiting::after {
+  animation: rotation-loop-after 2s linear infinite;
+}
+
+.aiMomentLoading::before,
+.aiMomentExiting::before {
+  animation: rotation-loop-before 2s linear infinite;
+  filter: blur(10px);
+  opacity: 0.7;
+}
+
+.aiMomentExiting::after,
+.aiMomentExiting::before {
+  animation-iteration-count: 1;
+}
+
+@keyframes rotation-loop-before {
+  0% {
+    background-size: 100% 100%;
+  }
+
+  50% {
+    --gradient-angle: 270deg;
+
+    background-position: 0% 20%;
+    background-size: 200% 300%;
+    opacity: 0.3;
+  }
+
+  100% {
+    background-size: 100% 100%;
+  }
+}
+
+@keyframes rotation-loop-after {
+  0% {
```

You can combine animation steps that do the same thing
```suggestion
  0%, 100% {
```